tests/run_all_tests.sh

```diff
@@ -56,7 +56,7 @@ fi
 
 # Run the full usual build process, checking to ensure there are no
 # closure compiler warnings / errors.
-run_test_command "build-debug" "npm run build-debug"
```

Comment:
Tested that this gives you the debug output? Just want to make sure it has the correct number of `--`s lol

Reply:
Yes, tested.  It took a bit of work to get Closure Compiler to generate a diagnostic, actually, now that we are suppressing most type errors…